### DISCUSSION TITLE
Can you have each repo in your github org stand up a website with this template?

### DISCUSSION THREAD

**Discussion by ADParedes** [alshedivat/al-folio#1271] [Q&A] [answered]
Hi.  I have followed the steps to create a github org website with URL [org name].github.io website.  It worked well! Now I am trying to create a new repo in my org using the same template.   Its not working. 

Shouldn't it be as simple cloning it into a new repo.  And then updating _config.yml to have a new base url?
```
url: "https://[org name].github.io" # the base hostname & protocol for your site
baseurl:  "/[new repo name]/"# the subpath of your site, e.g. /blog/
```
This didn't work btw but im trying to figure out why.  In git actions build baseline URL is giving me the Error of exit code 1.

**Comment by george-gca** [comment, marked as answer]
This should work as expected. In my case I have two GitHub urls: my [personal website](https://george-gca.github.io/) and [my template](https://george-gca.github.io/multi-language-al-folio/). You can check both sites' sources if you want.

Currently it looks like this:
| site | repo name | `baseurl` in _config.yml |
| --- | --- | --- |
| https://george-gca.github.io/ | [george-gca.github.io](https://github.com/george-gca/george-gca.github.io) | |
| https://george-gca.github.io/multi-language-al-folio/ | [multi-language-al-folio](https://github.com/george-gca/multi-language-al-folio) | /multi-language-al-folio |
